metals/src/main/scala/scala/meta/internal/metals/MetalsServerConfig.scala

```diff
@@ -106,6 +111,8 @@ object MetalsServerConfig {
           openFilesOnRenames = true,
           executeClientCommand = ExecuteClientCommandConfig.on,
           globSyntax = GlobSyntaxConfig.vscode,
+          isInputBoxEnabled = true,
+          isPickInputEnabled = true,
```

Comment:
As a follow-up to my previous comment, and similiar to what @tgodzik mentioned, if you rebase on master now, you'll see the pattern I followed in #1414. Moving forward for _most cases_, we shouldn't need to add new server properties, but rather list them under `ClientExperimentalCapabilities`.

Reply:
I've added client properies, but server ones are still there, cause `https://github.com/scalameta/metals-vscode/pull/207` is not being pushed.

Reply:
But that's because those properties already exist and are relied upon via clients that are already using them. This is a brand new one that we are introducing, so I'm not sure I understand why we also need to support this with client properties since nothing is relying on it? We can just only support it via `ClientExperimentalCapabilities`, and introduce it in vscode via experimentl.

Reply:
`isPickInputEnabled` removed now, and capability added to the client in https://github.com/scalameta/metals-vscode/pull/183. Though `isInputBoxEnabled = true` still in this place, I am not sure, if it should not be here now.

Reply:
Yea,  for now that's fine since that property already existed anyways. Once we release, and the the vs code pr gets merged, we can then remove the `isInputBoxEnabled` since then it will be set via experimental from vs code.